docs/source/llms/transformers/tutorials/text-generation/text-generation.ipynb

```diff
@@ -0,0 +1,530 @@
+    "\n",
+    "Once loaded, `sentence_generator` acts as a regular Python function that can be used to generate text based on the prompts provided to it. This function encapsulates the model's inference capabilities and allows us to seamlessly generate predictions without having to manually preprocess the input or postprocess the output and allows a fully encapsulated function that can be deployed anywhere."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "41e1a0c173744ceea33b1e9d282727a0",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Downloading artifacts:   0%|          | 0/20 [00:00<?, ?it/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2023/11/08 19:41:05 INFO mlflow.store.artifact.artifact_repo: The progress bar can be disabled by setting the environment variable MLFLOW_ENABLE_ARTIFACTS_PROGRESS_BAR to false\n"
+     ]
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "b5edb452d2e8466687e7f416ef1c5eee",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Loading checkpoint shards:   0%|          | 0/7 [00:00<?, ?it/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "sentence_generator = mlflow.pyfunc.load_model(model_info.model_uri)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Formatting Predictions for Tutorial Readability\n",
+    "Please note that the following function, `format_predictions`, is designed specifically for enhancing the readability of model predictions within this Jupyter Notebook environment. It **is not a standard component** of the model's inference pipeline and is **not necessary for the actual usage** of the model in production or other applications.\n",
+    "\n",
+    "The function takes a list of prediction strings and formats each one by:\n",
+    "\n",
+    "- Splitting the prediction text into individual sentences.\n",
+    "- Ensuring that each sentence is properly stripped of leading/trailing whitespace and ends with a period.\n",
+    "- Joining the sentences back together with newline characters for clear visual separation.\n",
+    "\n",
+    "This formatting is particularly useful in a notebook setting where we want to present the output in a more readable and presentable manner for demonstration purposes."
+   ]
+  },
```

Comment:
Do we need this explanation?

Reply:
Good call. Removed everything but the first 2 sentences (and shortened them).